index.html

```diff
@@ -69,6 +69,40 @@
                 </div>
             </div>
         </div>
+
+        <div id="tools-container">
+            <span class="title">
+                Tools
+            </span>
+            <div id="tools-container">
+                <div id="output-container">
+                    <div id="button-container">
+                        <button id="train-btn">Train</button><button id="analyze-btn">Analyze</button>
+                    </div> 
+                    <div id="tools-output">
+                        <table>
+                            <tbody>
+                                <tr>
+                                    <td>Accuracy:</td><td><span id="accuracy"></span></td>
+                                </tr>
+                                <tr>
+                                    <td>Runtime:</td><td>____________________</td>

```

Comment:
Therefore we can add value to the runtime here by adding the `<span>` tag, may be as follows:
```html
<td>Runtime:</td><td><span id="runtime-accuracy"></span></td>
```
and add a functionality in the JS file similar for handling the `accuracy`.